src/js/gi-bill-comparison-tool/components/Dropdown.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+
+class Dropdown extends React.Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      value: props.defaultOption
+    };
+  }
+
+  render() {
+    if (!this.props.visible) {
+      return null;
+    }
+    return (
+      <div id={this.props.identifier + '-form'} className={this.props.className}>
+        {this.renderLabel()}
+        <select id={this.props.identifier}
+            name={this.props.identifier.replace(/\-/g, '_')}
+            alt={this.props.alt}
+            onChange={(e) => { this.props.onChange(e) && this.setState({ value: event.target.value }); }}
+            defaultValue={this.props.defaultOption}>
+          {this.renderDropdownOptions(this.props.options)}
+        </select>
+      </div>
+    );
+  }
+
+  renderLabel() {
+    if (!this.props.showLabel) {
+      return null;
+    }
+    return this.props.children;
+  }
+
+  renderDropdownOptions(optionsArray) {
```

Comment:
Is there a reason for passing the options array as an array of strings, instead of an array of objects? It seems like it's making this more complicated.

If you passed in options arrays like this:

```
[
  { value: 'yes', label: 'Eligilble for Post-9/11 Gi Bill' },
  { value: 'no', label: 'Not Eligilble for Post-9/11 Gi Bill'}
]
```

This function can be one line:

`return optionsArray.map(({ value, label }) => <option key={value} value={value}>{label}</option>);`